docker/ubuntu-reportd/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:22.04
```

Comment:
Shouldn’t we base on `ubuntu:latest` so that we follow latest LTS automatically when rebuilding automatically (probably testing of the service using the docker image + CD in the future)?

It will be one place less you have to deal with when we have a release.

Reply:
In the past I've have had bad experiences with services using `:latest` so my natural inclination is to prefer explicit versions and then bump deliberately :) Do you have a strong opinion on this?

Reply:
it will ~potentially~ reduce maintenance ... so lets give it a try :) I'll need to update a few more pipelines (ubuntu-provision and app-center-ratings) to follow suit

Reply:
done!